qdrant-landing/content/articles/multitenancy.md

```diff
@@ -0,0 +1,189 @@
+preview_dir: /articles_data/multitenancy/preview
+small_preview_image: /articles_data/multitenancy/scatter-graph.svg
+weight: -101
+author: David Myriel
+date: 2024-02-06T13:21:00.000Z
+draft: false
+keywords:
+  - multitenancy
+  - custom sharding
+  - multiple partitions
+  - vector database
+---
+
+Whether you're building a fraud-detection analytics app, a RAG solution for e-commerce sites, or a healthcare portal for the government - you need to leverage a multitenant architecture and ensure access to your data is controlled. In the world of SaaS and large-scale enterprise solutions, this setup is the norm and it will considerably increase performance and lower your hosting costs. A single Qdrant cluster has the potential to support all of your customers, while each one of them would only see their own data. At times, if this data is location-sensitive, Qdrant gives you the option to divide your cluster by region or other criteria. 
+
+We are seeing the topics of multitenancy and distributed deployment pop-up daily on our [free support Discord](https://qdrant.to/discord) channel, which tells us that users are looking to scale Qdrant along with the rest of their ML setup. If you are currently mulling over your vector database implementation and are thinking of creating multiple collections for each user, read on. 
+
+[Multitenancy](https://qdrant.tech/documentation/guides/multiple-partitions/) is one of the most useful features Qdrant can offer, whereby you can partition one database instance for a vast number of users. Combining this with [Custom Sharding](https://qdrant.tech/documentation/guides/distributed_deployment/#user-defined-sharding) will result in an efficiently-partitioned architecture that further leverages the convenience of a single Qdrant cluster. This article will briefly explain the benefits and show how you can get started using both features.
+
+## Multitenancy: one collection, many users
+
+Multitenant architectures are almost required when scaling out your service and we recommend our multitenancy feature to most users. Under this setup you can upsert all your data to a single Qdrant collection, and then partition each vector via its payload. This means that all your users are leveraging the power of a single Qdrant cluster, but their data is still kept separate.
+
+**Figure 1:** Each individual vector is assigned a specific payload that denotes which user it belongs to. This is how a large number of different users can share a single Qdrant collection.
+![Qdrant Multitenancy](/articles_data/multitenancy/multitenancy-single.png)
+
+As a vector search engine, Qdrant is built to excel in multitenant environments. You should only create multiple collections when you have a limited number of users and you need isolation. Creating numerous collections may result in resource overhead and may cause dependencies. You always need to ensure that they do not affect each other in any way, including performance-wise. 
```

Comment:
To my mind, we should avoid encouraging anyone to create separate collections, even if the number of users is limited. We can use a single collection as long as the same embedding model generates vectors and when the data is homogenous (we can refer to SQL tables, which are also shared by multiple users/orgs).